dictionary/templates/dictionary/search.html

```diff
@@ -0,0 +1,28 @@
+{% extends "layout.html" %}
+{% load bootstrap3 %}
+
+{% block title %}Search{% endblock title %}
+
+
+{% block body %}
+    <form method="POST" action="" >
```

Comment:
If you wanted to change this to a GET method you'd change `method="POST"` to `method="GET"`